src/schema/v2/artist/verifiedRepresentatives.ts

```diff
@@ -0,0 +1,51 @@
+import { GraphQLFieldConfig, GraphQLList, GraphQLNonNull } from "graphql"
+import { ResolverContext } from "types/graphql"
+import { PartnerType } from "schema/v2/partner/partner"
+
+const VerifiedRepresentatives: GraphQLFieldConfig<
+  { _id: string },
+  ResolverContext
+> = {
+  type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(PartnerType))),
+  resolve: async (
+    { _id },
+    options,
+    { verifiedRepresentativesLoader, partnersLoader }
+  ) => {
+    if (!verifiedRepresentativesLoader) {
+      throw new Error("You are not authorized to perform this action.")
+    }
+    try {
+      const verifiedRepresentatives = await verifiedRepresentativesLoader(
+        { artist_id: _id },
+        options
+      )
+
+      if (!verifiedRepresentatives || !Array.isArray(verifiedRepresentatives)) {
+        throw new Error("Invalid response from verifiedRepresentativesLoader.")
+      }
+
+      const partnerIds = verifiedRepresentatives.map(
```

Comment:
It's up to you, you could modify the verified representatives endpoint to actually return some nested partner JSON, instead of just `partner_id`, which you then need to collect and do a bulk look-up on. This is fine though, that could always be changed later (as it won't change this schema shape to clients, it's fine to iterate on).